### PR TITLE
7.x 1.8

### PR DESCRIPTION
**Pull request by pc37utn**
**JIRA Ticket**: (link)
- Other Relevant Links (Google Groups discussion, related pull requests, Release pull requests, etc.)
# What does this Pull Request do?

  Updates screenshots of configuration of islandora_oai.  

A brief description of what the intended result of the PR will be and/or what problem it solves.
Updates the readme.
# What's new?

 Removes old screenshot,
 corrects some text in description,
 adds text for extra screen shots from documentation wiki.
# How should this be tested?

Compare the readme to the documentation wiki.
# Interested parties

@DonRichards @jordandukart

### CODE REVIEW

**Comment by DonRichards** [comment]
Ping @jordandukart

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## Introduction
 
-This module provides an implementation the Open Archives Initiative Protocol for Metadata Harvesting (OAI-PMH) for Islandora fedora repository with a solr index. By implementing the islanodra_oa module, you can expose content (its metadata) as an OAI-PMH repository. It will then be accessible by OAI harvesters For further OAI documentation. For more info, please see [this]( http://www.openarchives.org/OAI/openarchivesprotocol.html) link.
+This module provides an implementation of a data provider according to the Open Archives Initiative Protocol for Metadata Harvesting (OAI-PMH) for an Islandora fedora repository with a solr index. By implementing the islanodra_oai module, you can expose content (its metadata) as an OAI-PMH repository. It will then be accessible by OAI harvesters For further OAI documentation. For more info, please see [this]( http://www.openarchives.org/OAI/openarchivesprotocol.html) link.
```
Comment (DiegoPino) [inline README.md]: Please correct "islanodra_oai"

Comment (DiegoPino) [inline README.md]: "...OAI harvesters For further OAI documentation" something was left behind there.

File: README.md [hunk 2]
```diff
@@ -26,7 +26,31 @@ Install as usual, see [this](https://drupal.org/documentation/install/modules-th
...
+* Solr XACML role field - The site's Solr fields defining viewing permissions.
+* Solr hasModel field - The site's Solr field defining an object's content model.
+* Exclude Content Models - A list of content models, defined by their PID, to exclude from harvests. - if you exclude the collection content model, it disables the use of "sets" in OAi, as the name of the set is made by taking the PID of the collection and replacing the colon with an underscore.
+* Exclude objects within the "islandora" namespace
+* Append on dc.identifier.thumbnail to OAI_DCrequests? - this only has an effect on OAI_DC output.
+
+## Metadata Format
+
+This section allows you to configure the settings for the OAI-PMH'smetadata_prefix verb; Islandora uses XSL files to define the method for transforming your site's metadata datastreams into a format compatible with the OAI-PMH. Islandora OAI comes with two XSL files; they convert the MODS datastream of an object to either Electronic Thesis and Dissertation Metadata Standard format or Dublin Core format, which then can be served up to a harvester.
```
Comment (DiegoPino) [inline README.md]: `OAI-PMH'smetadata_prefix` needs a space

**Review by DiegoPino — COMMENTED** [review]
There seems to be some left overs of the previous readme + some proof reading missing

**Comment by pc37utn** [comment]
corrected

**Pull request merged**